### ISSUE TITLE
Tree and Tree Item - Ensure color is not the sole means of communicating information - (2036570423)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5620]
### Module: 
 38 Tree and Tree Item 

### Violation: 
 Ensure color is not the sole means of communicating information 

### WCAG Reference: 
- [1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) 

### Instance ID: 
 2036570423 

### Severity: 
 10 

### Description: 
 [Issue]
There are images in the "Child 1" controls that uses color alone to visually indicate selected state information.

[User Impact]
Users with color blindness will have difficulty identifying the colors used to convey this information.

[Code Reference] 
 ```
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 16 16" width="100%" xmlns="http://www.w3.org/2000/svg"><path d="M8 6a2 2 0 1 0 0 4 2 2 0 1 0 0-4z"></path></svg>
 ``` 

### Note: 
 [Suggestion]
Ensure color is not the sole means of communicating information. Color can be used as long as other visual indicators of the information are also present. Common additional visual indicators include underlining, bolding, additional on-screen text conveying the information, or change in shape or size.
 

### Media Type: 
 Color and Contrast 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Color and a symbol are depicted, so color is not the sole means of communication. Closing.
